src/Talis/Persona/Client/Signing.php

```diff
@@ -0,0 +1,120 @@
+<?php
+namespace Talis\Persona\Client;
+
+class Signing extends Base
```

Comment:
This class has been created to reduce the complexity in the Tokens class. For some reason the signing logic was bundled in Tokens even though they are unrelated topics.